client/query.go

```diff
@@ -213,23 +145,3 @@ func isQueryStoreWithProof(path string) bool {
 
 	return false
 }
-
-// parseQueryStorePath expects a format like /store/<storeName>/key.
-func parseQueryStorePath(path string) (storeName string, err error) {
```

Comment:
this is required for IBC proofs

Reply:
I can add this back but get unused (deadcode) linting error

Reply:
probably related to code removed in this PR. afaik this is not used in IBC proofs, @fedekunze do you see it somewhere?

Reply:
I don't think we need this. This is an internal func used by the verifyProof func which is only verifying proofs for the client. I don't see the connection to IBC